features/validators.feature

```diff
@@ -8,14 +8,15 @@ Feature: Data validators and output format
     **realType**  - required - Media Type of real data
     **expectedType** - optional - Media Type of expected (example) data used for validation
     **validator**  - optional - Data validator used for real and expected data comparison
-    **results** - required -  Validation result errors and warnings
+    **errors** - required -  Validation result errors and warnings
```

Comment:
good idea 👍

Reply:
I will describe my intention about internal/validation error handling in this pull request to drop some more light on these changes.